libs/dtm/dtm_recovery.py

```diff
@@ -277,6 +277,74 @@ def process_restart_with_delay(self, master_node, health_obj, pod_prefix, contai
 
         return True
 
+    def multi_process_restart_with_delay(self, master_node, health_obj, pod_prefix,
```

Comment:
Is it not possible to modify existing method of process restart to accommodate multiple processes? If its complicating existing method, ok to keep this.

Reply:
better to keep this separate.